src/ideal/ideal.jl

```diff
@@ -566,24 +566,6 @@ function jet(I::sideal, n::Int)
    return J
 end
 
-@doc Markdown.doc"""
```

Comment:
Do you really want to remove this?

Reply:
Yes, because the jacobian of an ideal does not make sense. 
One defines it in terms of a generating set. 
This was just a Singular convention, which I added at some point to have this functionality.